src/code/__osMalloc.c

```diff
@@ -243,9 +247,10 @@ void __osMalloc_FreeBlockTest(Arena* arena, ArenaNode* node) {
 
         while (iter < end) {
             if (*iter != BLOCK_UNINIT_MAGIC_32 && *iter != BLOCK_FREE_MAGIC_32) {
-                osSyncPrintf(
-                    VT_COL(RED, WHITE) "緊急事態！メモリリーク検出！ (block=%08x s=%08x e=%08x p=%08x)\n" VT_RST, node,
-                    start, end, iter);
+                osSyncPrintf(VT_COL(RED, WHITE) T("緊急事態！メモリリーク検出！ (block=%08x s=%08x e=%08x p=%08x)\n",
+                                                  "URGENT! MEMORY LEAK DETECTED! (block=%08x s=%08x e=%08x p=%08x)\n")
+                                 VT_RST,
+                             node, start, end, iter);
```

Comment:
the only difference between this and the similar messages above is the word for "found" and "detected". The word for "Emergency" is the same between them, and it probably should not be in all caps

I know we want to save translation cleanups for later, but I think we can at least fix glaring inconsistencies like this now

Reply:
I was trusting the thing as maybe the kanji had a more urgent meaning but sure I can change to lowercase, it's not like it makes sense from context for this check to be more meaningful than the others afaict